server/src/load_data.py

```diff
@@ -110,7 +111,7 @@ async def create_hnsw_index(
 
     application_field = TagField("application")
 
-    # Create index
+    # Create index (TODO: Compare the function `create_index()` above).
```

Comment:
Remove this comment. This line has nothing to do with the create_index above

Reply:
I've removed the `TODO` part of the comment. Or to remove it entirely?

Reply:
The To do